cli/azd/pkg/tools/azcli/azcli.go

```diff
@@ -284,13 +285,13 @@ type ListResourceGroupOptions struct {
 	// An optional tag filter
 	TagFilter *Filter
 	// An optional JMES path query to filter or project the result
-	JmesPathQuery *string
+	Filter *string
```

Comment:
Just wondering - are these still JMES queries or did the format change?

Reply:
No, they are not JMES queries via the REST API.  I will update the comment.